.gitignore

```diff
@@ -166,6 +166,9 @@ numpy/core/src/umath/simd.inc
 numpy/core/src/umath/struct_ufunc_test.c
 numpy/core/src/umath/test_rational.c
 numpy/core/src/umath/umath_tests.c
+numpy/core/src/umath/_umath_tests.dispatch.avx2.c
+numpy/core/src/umath/_umath_tests.dispatch.h
+numpy/core/src/umath/_umath_tests.dispatch.sse41.c
```

Comment:
added a few files to gitignore, maybe got added as part of https://github.com/numpy/numpy/pull/13516 ?